src/domain/app/layout/utilityComponents/PageMeta.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useEffect } from 'react';
+import { Helmet } from 'react-helmet-async';
+import { useTranslation } from 'react-i18next';
+import { useMatomo } from '@datapunt/matomo-tracker-react';
+
+import { getCurrentLanguage } from '../../../../common/translation/TranslationUtils';
+
+type Props = {
+  title?: string;
+  description?: string;
+};
+
+const PageMeta = ({
+  title = 'appName',
+  description = 'homePage.hero.descriptionText',
+}: Props) => {
+  const { i18n, t } = useTranslation();
+  const lang = getCurrentLanguage(i18n);
+
+  const translatedTitle =
+    title !== 'appName' ? `${t(title)} - ${t('appName')}` : t('appName');
+  const translatedDescription =
+    title !== 'homePage.hero.descriptionText'
+      ? t(description)
+      : t('homePage.hero.descriptionText');
+
+  // TODO: Switch to updateLocaleParam() from RouteUtils
+  // https://helsinkisolutionoffice.atlassian.net/browse/KK-223
+
+  // When fixing this, ensure that PageWrapper.test.tsx.snap has sensible href links.
+  const path = window.location.pathname.replace(`/${lang}/`, '');
```

Comment:
Should this comment be removed now that PageWrapper snap doesn't contain these lines.